etc/cpplint.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -e
 
-cpplint src/*.h src/*.cc
+cpplint src/*.hpp src/*.h src/*.cpp
```

Comment:
This comment will be very soon redundant, because #785 will be merged into stable-3.4, stable-3.4 will be merged into master, and then eventually this be rebased onto that master.